apps/metabase/0.1.0/README.md

```diff
@@ -0,0 +1,17 @@
+# Metabase Analytics tool
+
+This OAM application hosts Metabase Analytics Tool on Napptive with minimal configuration and authentication.
+
+Resources Requirement: 
+- CPU: 1vCPU
+- RAM: 4GB
```

Comment:
With this requirements te application fails,
`metabase` requires memory: "4Gi"
 `metabase-db` and `metabase-test-db` require memory: "512M" each one.
could you adjust this? I tried it with 3Gi to `metabase` and it worked, but I do not know the reason  whty you set 4Gi.

Reply:
Okay I'll update it